fix: Correct command substitutions in installation script

ShivamB25 opened the pull request.

TomSweeneyRedHat commented:
LGTM.  FWIW, I'm planning to keep this as "the file" and will be changing the one on github.com/containers/libpod to point to this file.

lsm5 commented:
> LGTM. FWIW, I'm planning to keep this as "the file" and will be changing the one on github.com/containers/libpod to point to this file.

sgtm

ShivamB25 committed:
fix: Correct command substitutions in installation script

The previous script had issues with command substitutions due to incorrect use of single quotes. This commit fixes the problem by using double quotes, allowing proper interpretation of `$(lsb_release -rs)` and `$(dpkg --print-architecture)`.
The installation script now correctly sets up the necessary keyrings and sources list to install Podman from the devel:kubic:libcontainers:unstable repository on Ubuntu.

ShivamB25 committed:
Merge pull request #1 from ShivamB25/ShivamB25-patch-1

fix: Correct command substitutions in installation script
